test/controllers/auth/AuthActionSpec.scala

```diff
@@ -103,18 +107,41 @@ class AuthActionSpec extends UnitSpec with OneAppPerSuite with MockitoSugar {
     }
   }
 
+  "A user with a confidence level 50 and a Nino" should {
```

Comment:
Is confidence level 50 definitely sufficient for this service for a PAYE user? Has this been confirmed with anyone?

Reply:
No. I've been looking for someone to talk to about auth requirements (as mentioned in the auth documentation). Our BA and Project Manager know nothing about this. Who should we speak to? Thx.